tests/testthat/test-renderer1-global-variables.R

```diff
@@ -6,33 +6,45 @@ viz <- list(scatter=ggplot()+
                          showSelected="Species",
                          data=iris))
 
+
 myScript <- 'myArray = [];
 for(var b in window) { 
   if(window.hasOwnProperty(b)) {myArray.push(b);} 
 }
-return myArray;'
+myArray;'
 
 getVariables <- function(){
-  vars <- remDr$executeScript(myScript)
+    
+    vars <- remDr$Runtime$evaluate(myScript,returnByValue = TRUE)$result$value
```

Comment:
please use consistent indentation (2 spaces for every indent, here there are too many)

Reply:
done